crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -270,26 +263,12 @@ impl BlobVec {
         self.initialize_unchecked(index, value);
     }
 
-    /// Forces the length of the vector to `len`.
-    ///
-    /// # Safety
-    /// `len` must be <= `capacity`. if length is decreased, "out of bounds" items must be dropped.
-    /// Newly added items must be immediately populated with valid values and length must be
-    /// increased. For better unwind safety, call [`BlobVec::set_len`] _after_ populating a new
-    /// value.
-    #[inline]
-    pub unsafe fn set_len(&mut self, len: usize) {
```

Comment:
Why remove the getters/setters? 
The fields are private so it's not a big deal, but we lose the ability to annotate safety and so won't get clippy lints encouraging safety comments when people touch the fields.

Reply:
The primary thing is that they're dead code. They're no longer used in any form as these types are entirely internal. We'd need to add some `allow(dead_code)` annotations.